pkg/init/cmd/service/prepare.go

```diff
@@ -0,0 +1,220 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/vishvananda/netlink"
+)
+
+const (
+	wgPath      = "/usr/bin/wg"
+	nsenterPath = "/usr/bin/nsenter-net"
+)
+
+// Note these definitions are from moby/tool/src/moby/config.go and should be kept in sync
+
+// Runtime is the type of config processed at runtime, not used to build the OCI spec
+type Runtime struct {
+	Mkdir      []string    `yaml:"mkdir" json:"mkdir,omitempty"`
+	Interfaces []Interface `yaml:"interfaces" json:"interfaces,omitempty"`
+	BindNetNS  string      `yaml:"bindNetNS" json:"bindNetNS,omitempty"`
```

Comment:
Might there be cause to bind other types of ns in the future (pid, etc?)? Perhaps make it like:
```
  bindns:
     - net
     - pid
```
Which I think is just `BindNS []string` here.